scripts/rollback-manual.sh

```
@@ -0,0 +1,53 @@
+}
+
+print_warning() {
+    echo -e "${YELLOW}[WARN]${NC} $1"
+}
+
+print_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+print_status "Finding latest backup..."
+LATEST_BACKUP=$(ssh $SERVER "cd $REMOTE_DIR && ls -t backups/*.tar.gz 2>/dev/null | head -1")
```

Shellcheck (0.10.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)